### ISSUE TITLE
Edit Party's Luck counter

### ISSUE THREAD

**Issue by aniila** [MikeyTheA/PokeRogueExtra#1]
Hi, I'm requesting the feature of being able to edit shiny luck, it is possible through the session's "scene" within the "party" section, hopefully this should be enough of a clue as I cannot elaborate more on the code (I'm not very versed in coding). Thank you

**Comment by asfjyrgfs** [comment]
By this do you mean the luck of a pokemon in your party when in a run? (example in the picture)
![image](https://github.com/MikeyTheA/PokeRogueExtra/assets/170099338/dd3300cd-85e8-4c48-a565-4e010729d364)

**Comment by aniila** [comment]
Yup that's right, it would be cool to be able to bring just one shiny in a run and get SSS luck.
